sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/package-info.java

```diff
@@ -3,8 +3,7 @@
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
 /**
- * Package containing the data models for KeyVaultAccessControlClient, KeyVaultBackupClient and
- * KeyVaultSettingsClient. The key vault client performs cryptographic key operations and vault operations against
- * the Key Vault service.
+ * Package containing the data models for KeyVaultAccessControlClient. The key vault client performs cryptographic key
```

Comment:
I havan't found a way of having this mention all 3 clients, maybe with a simple transformation...

Reply:
Currently, the only supported way to edit `package-info.java` is through customization.